pkg/frontend/admin_openshiftcluster_kubernetesobjects_test.go

```diff
@@ -110,21 +125,34 @@ func TestAdminKubernetesObjectsGetAndDelete(t *testing.T) {
 			resourceID:   fmt.Sprintf("/subscriptions/%s/resourcegroups/resourceGroup/providers/Microsoft.RedHatOpenShift/openShiftClusters/resourceName", mockSubID),
 			objKind:      "ConfigMap",
 			objNamespace: "openshift-project",
-			mocks: func(tt *test, openshiftClusters *mock_database.MockOpenShiftClusters, kactions *mock_kubeactions.MockInterface) {
+			mocks: func(tt *test, a *mock_adminactions.MockInterface, oc *mockdatabase.MockOpenShiftClusters, s *mockdatabase.MockSubscriptions) {
 				clusterDoc := &api.OpenShiftClusterDocument{
+					Key: tt.resourceID,
 					OpenShiftCluster: &api.OpenShiftCluster{
 						ID:   "fakeClusterID",
 						Name: "resourceName",
 						Type: "Microsoft.RedHatOpenShift/openshiftClusters",
 					},
 				}
+				subscriptionDoc := &api.SubscriptionDocument{
```

Comment:
This is needed now since a standard set of clients is created for all `adminactions`